src/hotspot/os/bsd/permitForbiddenFunctions_bsd.hpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024, 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+#ifndef OS_BSD_PERMITFORBIDDENFUNCTIONS_BSD_HPP
+#define OS_BSD_PERMITFORBIDDENFUNCTIONS_BSD_HPP
+
+#include "permitForbiddenFunctions_posix.hpp"
+
+#endif // OS_BSD_PERMITFORBIDDENFUNCTIONS_BSD_HPP
```

Comment:
I thought you were eliminating these in favor of #ifdef where included?

Reply:
Removed the forbid files, forgot the permit files.  Sigh!  I'll push a new commit after running tests.